<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealDataPlayerClass</class>
 <widget class="QMainWindow" name="RealDataPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RealDataPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblFileName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>841</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Re</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="realdataplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Load</sender>
   <signal>triggered()</signal>
   <receiver>RealDataPlayerClass</receiver>
   <slot>LoadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>RealDataPlayerClass</receiver>
   <slot>ResetButtonEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>RealDataPlayerClass</receiver>
   <slot>StartButtonEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LoadFile()</slot>
  <slot>StartButtonEntered()</slot>
  <slot>ResetButtonEntered()</slot>
 </slots>
</ui>
